File: find-unique-binary-string/find-unique-binary-string.java
class Solution {
    public String findDifferentBinaryString(String[] nums) {
        
        int n = nums.length;
        
        StringBuilder unique = new StringBuilder();
        
        for (int i = 0; i < n; ++i) {
            
            char ch = nums[i].charAt(i) == '0' ? '1' : '0';
            
            unique.append(ch);
        }
        
        return unique.toString();
    }
}

File: find-unique-binary-string/find-unique-binary-string.cpp
class Solution {
public:
    string findDifferentBinaryString(vector<string>& nums) {
        
        string unique = "";
        
        for (int i = 0; i < nums.size(); ++i) {
            
            char ch = nums[i][i] == '0' ? '1' : '0';
            
            unique += ch;
        } 
        
        return unique;
    }
};